app/src/main/java/org/gdg/frisbee/android/common/GdgListFragment.java

```diff
@@ -116,6 +125,18 @@ public void onDestroyView() {
     public void onListItemClick(ListView l, View v, int position, long id) {
     }
 
+    /**
+     * This method will be called when new data needs to be
+     * appended to the List. Subclasses should override.
+     *
+     * @param page              The ListView where the click happened
```

Comment:
There is something wrong with param definitions. totalItemsCount cannot be the view clicked.